static-site/src/pages/influenza.jsx

```diff
@@ -1,6 +1,5 @@
 import React from "react";
 import Helmet from "react-helmet";
-import yaml from "js-yaml";
 import ScrollableAnchor, { configureAnchors } from "react-scrollable-anchor";
```

Comment:
🐛 there's something going on with the setup of scroll anchors at the moment. In dev mode this crashes the page:
```
Uncaught TypeError: element is null
    getElementOffset scroll.js:14
    doesElementContainScrollTop scroll.js:29
    getBestAnchorGivenScrollLocation scroll.js:88
    getBestAnchorGivenScrollLocation scroll.js:86
...
```

In production mode this is limited to a console error & the page continues to work.

Reply:
I get this too but doesn't crash the page for me. It looks familiar so I it could be that it wasn't introduced here and I managed to ignore it before in the build-select branch if it wasn't causing crashes but just warnings in the console. Will see what's causing it / where it is introduced.

Reply:
Introduced in https://github.com/nextstrain/nextstrain.org/pull/278/commits/bafb08b0d68ea30d10fca9d49fb0d711cd5444c2
Fix is https://github.com/gabergg/react-scrollable-anchor/issues/43#issuecomment-538661917

Reply:
Will push the fix soon

Reply:
resolved in https://github.com/nextstrain/nextstrain.org/pull/279/commits/3fcb3d6b06290ee9a65d6554cd083674e3b95892